src/containers/Inspector.tsx

```diff
@@ -293,9 +372,21 @@ const Inspector: React.FC<IProps> = (props) => {
             src="https://github.com/open-rpc/design/raw/master/icons/open-rpc-logo-noText/open-rpc-logo-noText%20(PNG)/128x128.png" //tslint:disable-line
           />
           <Typography variant="h6" color="textSecondary">Inspector</Typography>
-          <IconButton onClick={handlePlayButton}>
-            <PlayCircle />
-          </IconButton>
+          {/* <Button
```

Comment:
what's this comment about?

Reply:
yep dead code. swapped it last minute for the icon button